### DISCUSSION TITLE
rút tiền

### DISCUSSION THREAD

**Discussion by nongdan111** [blockscout/blockscout#9847] [Q&A]
tôi rút tiền từ xterio chain sang bnb chain mà mã thông báo không về ví phải sử lý ra sao 
0x78c28cfbdcf842dede9122da151f5c102c366d3039f1a850b9fdd1f670fb2dd7

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
